build.gradle

```diff
@@ -102,6 +102,19 @@ dependencies {
     testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine"
 }
 
+sourceSets {
+    main {
+        java {
+            srcDirs = ['src/main/java', 'src/main/kotlin']
+        }
+    }
+    test {
+        java {
+            srcDirs = ['src/test/java', 'src/test/kotlin']
+        }
+    }
+}
+
```

Comment:
This is not in the right branch. Recommend you make this change in the dev branch and merge to the rest.